src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -461,4 +467,39 @@ private String convertTemplateProperties(String postmanUri) {
             return postmanUri.replaceAll("\\{\\{", "{").replaceAll("\\}\\}", "}");
         }
     }
+
+    private class PostmanImporterWorker implements Worker {
+        private String url;
+        private String postmanJson;
```

Comment:
``postmanJson`` => ``collectionJson``?

Reply:
@Alexander-Sviridov, I can change it, but it wasn't my creation.

Reply:
OK, in this case let's keep it as is.